components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -329,6 +352,13 @@
                     computed_value::T(Some(l.to_computed_value(context))),
             }
         }
+
+        #[inline]
+        fn to_specified_value(computed: &computed_value::T, context: &UncomputeContext) -> Self {
+            computed.0.map(|ref lop| {
+                SpecifiedValue::Specified(ToComputedValue::to_specified_value(lop, context))
+            }).unwrap_or(SpecifiedValue::Normal)
+        }
```

Comment:
(Might be worth pulling out this type out to a LengthOrPercentageOrNormal type in values.rs later.)